platform/build-faiss/build-faiss.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+git clone https://github.com/facebookresearch/faiss
+cd faiss
+
+# Configure paths and set environment variables
+export PATH=$PATH:$HOME/.local/bin:/usr/local/cuda/bin
+source /opt/intel/oneapi/setvars.sh
```

Comment:
This is worth comment. I would provide a reference, which should be sufficient:

https://www.intel.com/content/www/us/en/docs/oneapi/programming-guide/2023-1/use-a-config-file-for-setvars-sh-on-linux-or-macos.html